kpi/deployment_backends/base_backend.py

```diff
@@ -793,7 +798,8 @@ def _rewrite_json_attachment_urls(
             attachment['filename'] = os.path.join(
                 self.asset.owner.username,
                 'attachments',
-                submission['formhub/uuid'],
+                # KoboCAT accepts submissions even when they lack `formhub/uuid`
+                self.form_uuid or submission['formhub/uuid'],
```

Comment:
I want a 500 error here (`KeyError`) if neither the KoboCAT `backend_response` nor the submission have a form UUID.